### DISCUSSION TITLE
TracingContext: More than 300 spans required to create

### DISCUSSION THREAD

**Discussion by BritneyStars** [apache/skywalking#8533] [Q&A] [answered]
I found in my application, sometimes it is easy to create more than 300 spans. Because in my springboot crontab task, it operators mysql and elasticsearch loop many times ,when every time execute elasticsearch query DSL, it will create a span . Finally ,it will cause
  WARN 2018-08-15 19:04:36 TracingContext :  More than 300 spans required to create 
  java.lang.RuntimeException: Shadow tracing context. Thread dump.....
I want to known how to reslove it ,or shield some same spans because of elasticsearch query DSL ?

**Comment by wu-sheng** [comment, marked as answer]
It depends on yourself. You could remove some plugins or increase the config threshold over 300.

**Reply by BritneyStars** [reply]
ok, I known. Thx
